en/scripting/coding-setup.md

```diff
@@ -59,91 +57,88 @@ Cocos Creator 3.x 在创建项目时，项目目录下会自动生成一个 [tsc
+The above fields will set VS Code to exclude directories when searching, and file types hidden in the file list. Since `build`, `temp`, and `library` are all paths automatically generated when the editor is running, and will contain the content of the script we wrote, they should be excluded from the search. And each file in the `assets` directory will generate a `.meta` file. Generally speaking, there is no need to care about its content, just let the editor help manage these files.
 
-## VS Code 扩展使用
+## VS Code extension use
 
-Cocos Creator 在顶部菜单栏的 **开发者 -> VS Code 工作流** 中集成了 **添加编译任务** 和 **添加 Chrome Debug 配置** 功能，以便更好地辅助开发：
+Cocos Creator integrates the **Add Compilation Task** and **Add Chrome Debug Configuration** functions in **Developer -> VS Code Workflow** in the top menu bar to better assist development:
 
 ![vscode workflow](coding-setup/vscode-workflow.png)
 
-- **添加编译任务**：用于在 VS Code 中触发 Creator 的脚本编译，详情请参考下文 **使用 VS Code 激活脚本编译** 部分的内容。
+- **Add compilation task**: Used to trigger the script compilation of Creator in VS Code. For details, please refer to the content in the **Use VS Code to activate script compilation** section.
 
-- **添加 Chrome Debug 配置**：用于调试网页版游戏，详情请参考下文 **使用 VS Code 调试网页版游戏** 部分的内容。
+- **Add Chrome Debug configuration**: It is used to debug the web version of the game. For details, please refer to the following **Use VS Code to debug the web version of the game** section.
 
-### 使用 VS Code 激活脚本编译
+### Use VS Code to activate script compilation
 
-使用外部脚本编辑器修改项目脚本后，需要返回 Cocos Creator 以触发脚本编译。<br>
-我们在 Creator 中提供了 **添加编译任务** 功能，通过一个预览服务器的 API 向特定地址发送请求来激活 Creator 的编译，这样在外部脚本编辑器修改了项目脚本后，执行 **编译任务** 便可触发脚本编译，不需要返回 Cocos Creator。
+After using the external script editor to modify the project script, return to Cocos Creator to trigger script compilation. The **add compilation task** function is provided in Creator, which activates the compilation of Creator by sending a request to a specific address through the API of a preview server. After the external script editor modifies the project script, the **compilation task** is executed and can trigger script compilation without returning to Cocos Creator.
 
-#### 安装 cURL
+#### Install cURL
 
-首先需要确保操作系统中可以运行 [cURL 命令](https://curl.haxx.se/)，如果在 Windows 操作系统的命令行中运行 `curl` 提示找不到命令，则需要先安装 curl 到操作系统：
+First, make sure that the [cURL command](https://curl.haxx.se/) can be run in the operating system. If when running the `curl` command on the command line of the Windows operating system and the command cannot be found, install cURL to operating system:
 
-- 前往 <http://www.confusedbycode.com/curl/>
+- Visit this webpage for [help](http://www.confusedbycode.com/curl/)
 
-- 点击下图箭头所示的控件，完成人机身份验证（若无法正常显示控件，请科学上网）
+- Click the control shown by the arrow in the figure below to complete the human-machine identity verification (if the control cannot be displayed normally, please go online scientifically).
```

Comment:
```suggestion
- Click the control shown by the arrow in the figure below to complete the human-machine identity verification.
```

Reply:
"Going online scientifically" means using VPN or other ways to get over the GFW.

Reply:
ha ha, thanks. That is an interesting way to say VPN or GFW.

I had that as a note to ask you and forgot :-)